Makefile

```diff
@@ -26,6 +26,7 @@ SAIL_DEFAULT_INST += riscv_insts_dext.sail riscv_insts_cdext.sail
 SAIL_DEFAULT_INST += riscv_insts_zba.sail
 SAIL_DEFAULT_INST += riscv_insts_zbb.sail
 SAIL_DEFAULT_INST += riscv_insts_zbc.sail
+SAIL_DEFAULT_INST += riscv_insts_zbkb.sail
```

Comment:
Maybe separate Zbk* out from Zb* like you've done in riscv_sys_regs? It violates the alphabetical ordering, but I think it makes more sense? Not sure, but should probably at least be consistent one way or the other.

Reply:
Sure.